user/details/script.js

```diff
@@ -661,6 +659,45 @@ function showContent() {
   section3.classList.remove('hide');
 }
 
+function lockAccordianShowTooltip() {
```

Comment:
can we rename the function to `lockAccordiansForNonSuperUser`

Reply:
done, changed